demos/human_pose_estimation_demo/python/README.md

```diff
@@ -30,53 +19,55 @@ the argument, refer to **When to Reverse Input Channels** section of
 
 Running the application with the `-h` option yields the following usage message:
 ```
-python3 human_pose_estimation.py -h
-```
-The command yields the following usage message:
-```
-usage: human_pose_estimation.py [-h] -i INPUT [--loop] [-o OUTPUT]
-                                -m MODEL -at {ae,openpose}
-                                [--tsize TSIZE] [-t PROB_THRESHOLD] [-r]
-                                [-d DEVICE] [-nireq NUM_INFER_REQUESTS]
-                                [-nstreams NUM_STREAMS]
-                                [-nthreads NUM_THREADS] [-loop LOOP]
-                                [-no_show] [-u UTILIZATION_MONITORS]
+usage: human_pose_estimation_demo.py [-h] -m MODEL -at {ae,openpose} -i INPUT
+                                     [--loop] [-o OUTPUT]
+                                     [-limit OUTPUT_LIMIT] [-d DEVICE]
+                                     [-t PROB_THRESHOLD] [--tsize TSIZE]
+                                     [-nireq NUM_INFER_REQUESTS]
+                                     [-nstreams NUM_STREAMS]
+                                     [-nthreads NUM_THREADS] [-no_show]
+                                     [-u UTILIZATION_MONITORS] [-r]
 
 Options:
   -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model.
+  -at {ae,openpose}, --architecture_type {ae,openpose}
+                        Required. Specify model' architecture type.
   -i INPUT, --input INPUT
                         Required. An input to process. The input must be a
-                        single image, a folder of images, video file or camera id.
+                        single image, a folder of images, video file or camera
+                        id.
   --loop                Optional. Enable reading the input in a loop.
   -o OUTPUT, --output OUTPUT
                         Optional. Name of output to save.
-  -m MODEL, --model MODEL
-                        Required. Path to an .xml file with a trained model.
-  -at {ae,openpose}, --architecture_type {ae,openpose}
-                        Required. Type of the network, either "ae" for
-                        Associative Embedding or "openpose" for OpenPose.
+  -limit OUTPUT_LIMIT, --output_limit OUTPUT_LIMIT
+                        Optional. Number of frames to store in output. If -1
```

Comment:
Please, update the `limit` accordingly the other demos (from -1 to 0).

Reply:
Good catch